Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use a base image with platform specification
+FROM --platform=$BUILDPLATFORM debian:bookworm-slim
+
+# Define the arguments for Atmos version and platforms
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+ARG ATMOS_VERSION
+
+SHELL ["/bin/bash", "-c"]
+# Update the package list and install curl and git
+RUN apt-get update && apt-get install -y curl git
+
+# Install the Cloud Posse Debian repository
+RUN curl -1sLf 'https://dl.cloudsmith.io/public/cloudposse/packages/cfg/setup/bash.deb.sh' | bash
+
+# Install OpenTofu
+RUN curl -1sSLf 'https://get.opentofu.org/install-opentofu.sh' | bash -s -- --root-method none --install-method deb
+
+# Install Kustomize binary (required by Helmfile)
+RUN curl -1sSLf "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash -s -- /usr/local/bin
+
+# Install toolchain used with Atmos
+RUN apt-get -y install terraform kubectl helmfile helm
```

Comment:
This installation method may also be problematic for `arm` users as the HashiCorp APT server has packages only for the `amd64` architecture. On `non-amd64` systems, a `.zip` install is required

Reply:
Isn't that solved by this?

```
FROM --platform=$BUILDPLATFORM debian:bookworm-slim
```

Reply:
...as a multi-arch build

Reply:
Will mark this resolved and see if it comes back to bite us.